scripts/constituents.py

```diff
@@ -414,22 +397,22 @@ def constituent_module_name(self):
     def num_consts_funcname(self):
         """Return the name of the function which returns the number of
         constituents for this suite."""
-        return "{}_num_consts".format(self.name)
+        return f"{self.name}_num_consts"
 
     def const_name_subname(self):
         """Return the name of the routine that returns a constituent's
            given an index"""
```

Comment:
I think a word is maybe missing here (?):

```suggestion
        """Return the name of the routine that returns a constituent's
           standard name given an index"""
```

Reply:
fair point!